Packages/com.unity.inputsystem/InputSystem/Actions/InputBinding.cs

```diff
@@ -841,6 +843,18 @@ public bool Matches(InputBinding binding)
         {
             return Matches(ref binding);
         }
+        
+        internal bool MatchesMask(ref InputBinding binding)
+        {
+            var appendix = path.Substring(path.Length-2,path.Length-(path.Length-2));
+            if (appendix == wildcard)
+            {
+                var before = path.Substring(0, path.Length - 2);
+                if (binding.path != null && binding.path.Contains(before))
```

Comment:
Is Contains a bit too loose. Should this be StartsWith ?

Reply:
see comment below